model/distro/distro.go

```diff
@@ -100,6 +106,13 @@ var ValidBootstrapMethods = []string{
 	BootstrapMethodUserData,
 }
 
+// ValidCommunicationMethods includes all recognized host communication methods.
+var ValidCommunicationMethods = []string{
```

Comment:
I think that this can be private because we're only really interacting with it via the public method.

Reply:
Changed to unexported.